package.json

```diff
@@ -7,16 +7,16 @@
   "repository": "googleapis/nodejs-paginator",
   "scripts": {
     "test": "c8 mocha build/test",
-    "lint": "gts check",
-    "clean": "gts clean",
     "compile": "tsc -p .",
     "fix": "gts fix",
+    "prelint": "cd samples; npm link ../; npm i",
+    "lint": "gts check",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
     "posttest": "npm run lint",
     "docs": "compodoc src/",
     "presystem-test": "npm run compile",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
```

Comment:
@bcoe do we do this in other places?

Reply:
all that should be required is this:

```
"samples-test": "cd samples/ && npm link ../ && npm test && cd ../"
```

which is what we use in other libraries, `npm link ../` should perform the install.

Reply:
FYI: The couple libraries I converted use `samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../`, like https://github.com/googleapis/nodejs-monitoring/blob/master/package.json#L32